client/app/visualizations/table/Editor/editor.less

```diff
@@ -1,11 +1,15 @@
 .table-visualization-editor-columns {
+  .ant-collapse {
+    background: transparent;
+  }
+
   .ant-collapse-item {
     background: #ffffff;
```

Comment:
This would need `z-index: 1` as well.
Maybe give it to `< SortableItem>` instead?

Reply:
I'm not sure why, but in this case `z-index` not needed - everything works fine

Reply:
The blue tab underline!

<img width="593" alt="Screen Shot 2019-10-01 at 15 49 37" src="https://user-images.githubusercontent.com/486954/65963305-2b3faf80-e463-11e9-8aff-97939d355c22.png">

Reply:
Omg :smile: Okay, it's a good reason for `z-index` :slightly_smiling_face: